lib/hutch/worker.rb

```diff
@@ -103,5 +103,10 @@ def error_acknowledgements
     private
 
     attr_accessor :setup_procs
+
+    def consumer_tag(queue)
+      prefix = Hutch::Config[:consumer_tag_prefix]
+      queue.channel.generate_consumer_tag(prefix)
```

Comment:
Ah, it _does_ exist, as another way of expressing it: https://github.com/ruby-amqp/march_hare/pull/93/files